src/index.ts

```diff
@@ -11,6 +11,7 @@ import { Base58 } from '@ethersproject/basex'
 import { toUtf8Bytes } from '@ethersproject/strings'
 import { EthrDidController, interpretIdentifier, REGISTRY } from 'ethr-did-resolver'
 import { Resolvable } from 'did-resolver'
+import { address, MetaSignature } from 'ethr-did-resolver/lib/helpers'
```

Comment:
Looks like this import is failing. These types are simple enough to be redefined here until they get exported from the resolver package:

```suggestion

export type address = string
export type MetaSignature = {
  sigV: number
  sigR: bytes32
  sigS: bytes32
}
```